cdap-ui/app/features/dashboard/templates/staticdashboard.html

```diff
@@ -0,0 +1,22 @@
+<div class="row" ng-controller="WidgetColCtrl">
+  <div ng-class="{ 'col-xs-12': colWidth.fullWidth, 'col-xs-12 col-md-6': colWidth.oneThird }"
```

Comment:
This is a static dashboard right? Do we want to have an option of changing the layout here? Can't it be always 2 column or 4 column?